test/run.d

```diff
@@ -260,17 +261,17 @@ string[string] getEnvironment()
         env["OBJ"] = ".obj";
         env["DSEP"] = `\\`;
         env["SEP"] = `\`;
-        auto druntimePath = environment.get("DRUNTIME_PATH", `..\..\druntime`);
-        auto phobosPath = environment.get("PHOBOS_PATH", `..\..\phobos`);
+        auto druntimePath = environment.get("DRUNTIME_PATH", testPath(`..\..\druntime`));
+        auto phobosPath = environment.get("PHOBOS_PATH", testPath(`..\..\phobos`));
         env["DFLAGS"] = `-I%s\import -I%s`.format(druntimePath, phobosPath);
         env["LIB"] = phobosPath;
 
         // auto-tester might run the testsuite with a different $(MODEL) than DMD
         // has been compiled with. Hence we manually check which binary exists.
         // For windows the $(OS) during build is: `windows`
-        int dmdModel = "../generated/windows/%s/64/dmd%s".format(build, exe).exists ? 64 : 32;
+        int dmdModel = testPath(format(`..\generated\windows\%s\64\dmd%s`, build, exe)).exists ? 64 : 32;
         env.getDefault("MODEL", dmdModel.text);
-        env["DMD"] = "../generated/windows/%s/%d/dmd%s".format(build, dmdModel, exe);
+        env["DMD"] = testPath(format(`..\generated\windows\%s\%d\dmd%s`, build, dmdModel, exe));
```

Comment:
Looks like these changes are the only ones needed (this was just copied/translated from the makefile which still assumes that it's called from a posix-like shell, e.g. like the git shell on Windows).